db/migrate/20170901061817_add_columns_to_wind_turbine_vibration_measurement.rb

```diff
@@ -0,0 +1,13 @@
+class AddColumnsToWindTurbineVibrationMeasurement < ActiveRecord::Migration[5.0]
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?